openrtb_ext/imp_mobfoxpb.go

```diff
@@ -0,0 +1,6 @@
+package openrtb_ext
+
```

Comment:
Since you're using JSON parser to parse the `tagID` value, you don't really need this struct here as it's not being used anywhere